duckduckgo/iOS
# Fix autofill auto-disable for first time users

Task/Issue URL: https://app.asana.com/0/414709148257752/1202526099545584/f
CC: @GioSensation @THISISDINOSAUR 

**Description**:
Fix issue where autofill was being disable once the user denied the save modal 3x.
This should only happen if the user disables 3x in a row for a first time user.

**Steps to test this PR**:

1. on a brand new app installation go to a website like https://news.ycombinator.com/login
2. Fill username/password
3. Reject the autofill modal to save it
4. Do the above 4x
5. On the 4th time, no prompt should be displayed to the user
6. Check under settings/autofill if autofill is disabled, it should

------
1. on a brand new app installation go to a website like https://news.ycombinator.com/login
2. Fill username/password
3. Reject the autofill modal to save it
4. Repeat the above, but on step 3 save the password instead of rejecting
5. Repeat steps 1-3 3x more (not saving it)
6. Autofill should be always enabled and prompting to save 
7. Check under settings/autofill if autofill is enabled, it should

**OS Testing**:



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
